2.4/contrib/etc/mongodb.conf

```diff
@@ -0,0 +1,21 @@
+# mongodb.conf
+
+port = 27017
+#fork = true
+pidfilepath = /opt/rh/mongodb24/root/var/run/mongodb/mongodb.pid
+dbpath = /var/lib/mongodb
+
+# Disable data file preallocation.
+noprealloc = true
```

Comment:
I've set the same parameter that we are using in v2 as a starting point. What I thought about is, if there is some way how could be supply apps with user custom config, so we could avoid creating and documenting tons of env vars.